sympy/stats/frv.py

```diff
@@ -375,12 +372,120 @@ class SingleFinitePSpace(SinglePSpace, FinitePSpace):
     def domain(self):
         return SingleFiniteDomain(self.symbol, self.distribution.set)
 
+    @property
+    def _is_symbolic(self):
+        """
+        Helper property to check if the distribution
+        of the random variable is symbolic.
+        """
+        return isinstance(self.domain.set, Intersection)
```

Comment:
Why does this mean that it is symbolic? Also you need to come up with a better term to describe this. The doc string here is confusing.

Reply:
Right, I will introduce a better check here. I realised that this uses a very special property of distributions which can change in future, will introduce a general check.